### PR TITLE
tasks: Documentation updates

### PR DESCRIPTION
**Pull request by martinpitt**

### CODE REVIEW

File: tasks/README.md [hunk 1]
```diff
@@ -249,8 +248,8 @@ the scheduling for free, and is really easy to set up.
    up to 10 seconds and can fail, and HTTP requests ought to be replied fast
    and reliably.
 
- * Then all real test/release/etc. worker bots also connect to the same AMQP
-   container (locally through the service or remotely through the route).
+ * Then all worker bots also connect to the same AMQP container (locally
+ * through the service or remotely through the route).
```
Comment (marusak) [inline tasks/README.md:252]: This bullet seems odd here

File: tasks/README.md [hunk 2]
```diff
@@ -273,13 +273,13 @@ the scheduling for free, and is really easy to set up.
    infrastructure), executes it, publishes the log, updates the GitHub status,
    and finally acks the queue item.
 
-# Using with toolbox
+# Using with toolbx
```
Comment (marusak) [inline tasks/README.md:276]: Lol, thought this was typo :)

File: tasks/README.md (hunk 1 above)
Comment (martinpitt) [reply tasks/README.md]: Thanks! Overzealous auto-formatting in vim. Fixed.

File: tasks/README.md (hunk 2 above)
Comment (martinpitt) [reply tasks/README.md:276]: Heh, it seems to be the current naming pattern. Presumably to make these more google-able?

**Review by marusak — APPROVED** [review]

**Pull request merged**